From 78f48e26dccae9c8db8ca19f902bfe17813aec4b Mon Sep 17 00:00:00 2001
From: Munehisa Kamata <kamatam@amazon.com>
Date: Fri, 17 Mar 2017 17:17:29 +0000
Subject: drivers/amazon: add block device drivers support

This is a place holder for block device driver updates.

Signed-off-by: Munehisa Kamata <kamatam@amazon.com>
Reviewed-by: Cristian Gafton <gafton@amazon.com>
Reviewed-by: Guru Anbalagane <guruanb@amazon.com>

CR: https://cr.amazon.com/r/6834648/
---
 drivers/amazon/Makefile       | 1 +
 drivers/amazon/block/Makefile | 3 +++
 2 files changed, 4 insertions(+)
 create mode 100644 drivers/amazon/block/Makefile

diff --git a/drivers/amazon/Makefile b/drivers/amazon/Makefile
index fc5f70d..bdb1d0d 100644
--- a/drivers/amazon/Makefile
+++ b/drivers/amazon/Makefile
@@ -2,3 +2,4 @@
 # Amazon Driver Updates
 #
 obj-$(CONFIG_AMAZON_DRIVER_UPDATES)	+= net/
+obj-$(CONFIG_AMAZON_DRIVER_UPDATES)	+= block/
diff --git a/drivers/amazon/block/Makefile b/drivers/amazon/block/Makefile
new file mode 100644
index 0000000..5298efe
--- /dev/null
+++ b/drivers/amazon/block/Makefile
@@ -0,0 +1,3 @@
+#
+# Amazon Driver updates
+#
-- 
2.7.5

